patches/minecraft/net/minecraft/entity/player/EntityPlayer.java.patch

```diff
@@ -589,6 +591,11 @@
 +        return capability == net.minecraftforge.items.CapabilityItemHandler.ITEM_HANDLER_CAPABILITY || super.hasCapability(capability, facing);
 +    }
 +
++    private Integer spawnDimension;
```

Comment:
This field should be marked  `@Nullable`, otherwise everything looks good to me.